backend/e4h-services/ingestion-service/app/api/endpoints/file_ingestion.py

```
@@ -318,15 +318,13 @@ async def upload_facilities_with_workstream(
-@router.post('/facilityWithSupervisors',
-             summary='Upload and process facility with supervisors Excel file',
+@router.post('/facilityWithStaff',
+             summary='Upload and process facility with staff Excel file',
              response_description="Returns processed Excel file with validation results")
-async def upload_facility_with_supervisors_excel_sheet(
-        facility_with_supervisors: UploadFile = File(
-            description="Excel file containing facility with supervisors data"),
-        facility_sheet: str = Form(default="Facilities_Supervisors",
+async def upload_facility_with_staff_excel_sheet(
+        facility_with_staff: UploadFile = File(
+            description="Excel file containing facility with staff data"),
```

Ruff (0.11.9)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)